src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.module.css

```diff
@@ -0,0 +1,43 @@
+.text {
+  font-size: 50px;
+  font-style: var(--font-family-merriweather);
+  text-align: center;
+}
+
+.bleedWrapper {
+  display: grid;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
```

Comment:
You're missing a `grid-template-columns` definition on the non-mobile version of this element, and I think this is causing the component to render very strangely in Storybook. This is what I see in a desktop view (it's off-center):

<img width="1693" alt="Screen Shot 2020-10-18 at 7 35 16 PM" src="https://user-images.githubusercontent.com/1002748/96395342-4322c800-1179-11eb-8973-2296b3dff069.png">

And this is what I see on mobile:

<img width="455" alt="Screen Shot 2020-10-18 at 7 36 55 PM" src="https://user-images.githubusercontent.com/1002748/96395360-4ddd5d00-1179-11eb-9cd4-b42a8a3bcd9b.png">